DSP_Plugins.GalacticScale/Scripts/PatchPlanetSize/PatchOnPlanetModelingManager.cs

```diff
@@ -65,8 +66,62 @@ public class PatchOnPlanetModelingManager : MonoBehaviour {
 
 
             //data.heightData = scaledHeightData;
-
-            if (___currentModelingStage == 2) {
+            if (___currentModelingStage == 1) {
+                ___tmpPlanetGameObject = new GameObject(planet.displayName);
+                ___tmpPlanetGameObject.layer = 31;
+                PlanetSimulator sim = ___tmpPlanetGameObject.AddComponent<PlanetSimulator>();
+                GameMain.universeSimulator.SetPlanetSimulator(sim, planet);
+                ___tmpPlanetGameObject.transform.localPosition = Vector3.zero;
+                ___tmpPlanetBodyGameObject = new GameObject("Planet Body");
+                ___tmpPlanetBodyGameObject.transform.SetParent(___tmpPlanetGameObject.transform, worldPositionStays: false);
+                ___tmpPlanetBodyGameObject.layer = 31;
+                ___tmpPlanetReformGameObject = new GameObject("Terrain Reform");
+                ___tmpPlanetReformGameObject.transform.SetParent(___tmpPlanetBodyGameObject.transform, worldPositionStays: false);
+                ___tmpPlanetReformGameObject.layer = 14;
+                MeshFilter meshFilter = ___tmpPlanetReformGameObject.AddComponent<MeshFilter>();
+                ___tmpPlanetReformRenderer = ___tmpPlanetReformGameObject.AddComponent<MeshRenderer>();
+                meshFilter.sharedMesh = Configs.builtin.planetReformMesh;
+                Material[] sharedMaterials = new Material[2] { planet.reformMaterial0, planet.reformMaterial1 };
+                ___tmpPlanetReformRenderer.sharedMaterials = sharedMaterials;
+                ___tmpPlanetReformRenderer.receiveShadows = false;
+                ___tmpPlanetReformRenderer.lightProbeUsage = LightProbeUsage.Off;
+                ___tmpPlanetReformRenderer.shadowCastingMode = ShadowCastingMode.Off;
+                float num = (planet.realRadius + 0.2f + planet.realRadius / 8000f) * 2f;
```

Comment:
We may want to add a comment to this line, that this section was changed from
`planet.realRadius + 0.2f + 0.025`
to
`planet.realRadius + 0.2f + planet.realRadius / 8000f`
to adapt the foundation to the planet sizes.
Otherwise we would wonder why this whole block is here later on.

Reply:
ooooooh so that was it